### DISCUSSION TITLE
How do I make a secret edit link?

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#73] [Ask a question] [answered]
Now it works like this: domain.com/folder-name-01/**edit**/page-name-slug

I think it will be better to do: domain.com/folder-name-01/page-name-slug**---**
No matter left part slug is consist of, just add the "---" after any url and you can edit any page.

Very easy to remember and to use. No need special link on page at all.
It was realised in s2cms.ru and it very cool and simple feature.

**Comment by markseuffert** [comment, marked as answer]
Thanks for your feedback. 

It's a nice idea, you could add it to your `.htaccess` if you want a similar feature in Yellow.

```
RewriteCond %{REQUEST_URI} !^/edit/
RewriteRule ^(.*)---$ /edit/$1 [R=301,L]
```

**Comment by i-do** [comment]
Great! Thanks a lot! 
But it's doesn't work. "File not found" error.

**Comment by markseuffert** [comment]
Can you send me a link?

**Comment by markseuffert** [comment]
Looks like you are running Apache web server. Here's an `.htaccess` that works on my server:

```
<IfModule mod_rewrite.c>
RewriteEngine on

# Secret edit link

RewriteCond %{REQUEST_URI} !^/edit/
RewriteRule ^(.*)---$ /edit/$1 [R=301,L]

# Yellow dynamic pages

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^ yellow.php [L]
DirectoryIndex index.html yellow.php
RewriteRule ^(cache|content|system)/ error [L]

# Yellow static pages

RewriteCond %{REQUEST_FILENAME} -f
RewriteRule ^[^\.]+$ - [T=text/html,L]
ErrorDocument 404 /error.html
</IfModule>
```

**Comment by i-do** [comment]
Sorry, it's my mistake. I've add rules after closed IfModule tag, of cause it's doesn't work :)
Thanks for your help!
